.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "loopback",
+  "rules": {
+    "max-len": ["error", 110, 4, {
+      "ignoreComments": true,
+      "ignoreUrls": true,
+      "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
+    }],
+    // NOTE(bajtos) we should eventuall remove this override
+    // and fix all of those 100+ violations
+    "one-var": "off"
```

Comment:
This is a bit of a shortcut I took in order to get this landed reasonably soon. There were already too many changes needed to get eslint to pass (well over 400), these extra 100+ were simply too much to do in one go.